test/fixtures/programmatic/function/stream.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+// eslint-disable-next-line no-undef
```

Comment:
Instead of doing that, let's maybe add `awsLambda` as global into ESLint config, for the lambda fixtures.

ESlint config is maintained in `package.json`